lib/dry/view/default_renderer.rb

```diff
@@ -0,0 +1,9 @@
+module Dry
+  module View
+    class DefaultRenderer
+      def method_missing(name, *args, &block)
+        raise "No renderer provided"
+      end
```

Comment:
I reckon I'd feel better if we were returning a custom exception class here, maybe `MissingRendererError`. You OK to give that a go?

Reply:
Yep! That sounds good, was also thing we could add some info about the call site if needed, but I think the stack trace should suffice here.